files/8.1/debug/entrypoint.sh

```diff
@@ -7,7 +7,6 @@
 cat << EOF > /usr/local/etc/php/conf.d/20-xdebug.ini
 xdebug.idekey = PHPSTORM
 xdebug.mode = debug
-xdebug.start_with_request = 1
```

Comment:
This should be done for the PHP 8.0 debug entrypoint, too. 
I don't think it's necessary for PHP 7.4, as that's still using Xdebug 2.